lib/wanda_web/schemas/execution_response.ex

```diff
@@ -0,0 +1,58 @@
+defmodule WandaWeb.Schemas.ExecutionResponse do
+  @moduledoc """
+  Execution item response API spec
+  """
+
+  require OpenApiSpex
+
+  alias OpenApiSpex.Schema
+
+  alias WandaWeb.Schemas.ExecutionResponse.CheckResult
+
+  OpenApiSpex.schema(%{
+    title: "ExecutionResponse",
+    description: "The representation of an execution, it may be a running or completed one",
+    type: :object,
+    properties: %{
+      execution_id: %Schema{type: :string, format: :uuid, description: "Execution ID"},
+      group_id: %Schema{type: :string, format: :uuid, description: "Group ID"},
+      status: %Schema{
+        type: :string,
+        enum: ["running", "completed"],
+        description: "The status of the current execution"
+      },
+      started_at: %Schema{
+        type: :string,
+        format: :"date-time",
+        description: "Execution start time"
+      },
+      completed_at: %Schema{
+        type: :string,
+        nullable: true,
+        format: :"date-time",
+        description: "Execution completion time"
+      },
+      result: %Schema{
+        type: :string,
+        enum: ["passing", "warning", "critical", "unknown"],
```

Comment:
As put in other comment, I'm not convinced with the `unknown` value here

Reply:
removed and made nullable